packages/shorebird_cli/test/src/command_runner_test.dart

```diff
@@ -274,6 +271,35 @@ Run ${lightCyan.wrap('shorebird upgrade')} to upgrade.'''),
       });
     });
 
+    group('on command failure', () {
+      group('when running with --verbose', () {
+        setUp(() {
+          when(() => logger.level).thenReturn(Level.verbose);
+        });
+
+        test('does not suggest running with --verbose', () async {
+          // This will fail due to missing scoped overrides.
+          // Note: the --verbose flag is here for illustrative purposes only.
+          // Because logger is a mock, setting the log level in code does
+          // nothing.
+          await runWithOverrides(
+            () => commandRunner.run(['release', 'android', '--verbose']),
+          );
+          verifyNever(() => logger.info(any(that: contains('--verbose'))));
+        });
+      });
+
+      group('when running without --verbose', () {
+        test('suggests using --verbose flag', () async {
+          // This will fail due to missing scoped overrides.
```

Comment:
Same here I'm not sure I understand what the comment means.

Reply:
Same as above